src/main/resources/liquibase/workbench_changelog/19_4_0.xml

```diff
@@ -261,4 +261,62 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="cuenyad" id="v19.4.0-8">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<sqlCheck expectedResult="1">
+					SELECT count(*) FROM permission WHERE name = 'MANAGE_GERMPLASM'
+				</sqlCheck>
+				<sqlCheck expectedResult="0">
+					SELECT count(*) FROM permission WHERE name = 'MG_ADD_ENTRIES_TO_LIST'
+				</sqlCheck>
+			</and>
+		</preConditions>
+		<comment>
+			Add MG_ADD_ENTRIES_TO_LIST permission
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+
+			SET @permission_id = (SELECT permission_id FROM permission where name = 'MANAGE_GERMPLASM');
+
+			INSERT INTO permission (`name`, `description`, `parent_id`)
+			VALUES ('MG_ADD_ENTRIES_TO_LIST', 'Add to list', @permission_id);
+
+			SET @mg_add_entries_to_list_permission_id = (SELECT permission_id FROM permission where name = 'MG_ADD_ENTRIES_TO_LIST');
+
+			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('1', @mg_add_entries_to_list_permission_id, '1');
+			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('2', @mg_add_entries_to_list_permission_id, '1');
+			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('3', @mg_add_entries_to_list_permission_id, '1');
+
+		</sql>
+	</changeSet>
+	<changeSet author="cuenyad" id="v19.4.0-9">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
```

Comment:
@cuenyad Let's review this on Monday

Reply:
Adding MG_ADD_ENTRIES_TO_LIST permission in role_permission with SEARCH_GERMPLASM permission.